tests/wallet_tools.py

```diff
@@ -30,6 +30,7 @@
 class WalletTool:
     next_address = 0
     pubkey_num_lookup: Dict[bytes, uint32] = {}
+    puzzle_pk_cache: Dict[bytes32, PrivateKey] = {}
```

Comment:
```suggestion
    puzzle_sk_cache: Dict[bytes32, PrivateKey] = {}
```

By cryptography programmer conventions, this variable should be named `puzzle_sk_cache`

Reply:
I wanted to keep my changes focused on just this issue of mine. Also, by the same standards, the `PrivateKey` type should be called `SecretKey` :)